src/cf-forms/usage.md

```diff
@@ -1,12 +1,34 @@
-# Forms
+The cf-forms component provides extensions to the basic form styles for Capital Framework.
+
+[`cf-core`](../cf-core), ['cf-buttons'](../cf-buttons), and ['cf-icons'](../cf-icons) components are all dependencies of this component."
 
 > NOTE: If you use `cf-forms.less` directly,
   be sure to run the file through
   [Autoprefixer](https://github.com/postcss/autoprefixer),
   or your compiled Capital Framework CSS will
   not work perfectly in older browsers.
 
-## Vars
+## Table of contents
+
+- [Variables](#variables)
+    - [Color variables](#color-variables)
```

Comment:
This is minor, but can you include the line about the colors being from the U.S. Web Design Standards and comments with the colors to be consistent with the rest?